guides/common/modules/proc_upgrading-el7-to-el8.adoc

```diff
@@ -164,6 +232,18 @@ If this happens, do the following to clean up packages that cannot automatically
 ----
 # journalctl -u leapp_resume -f
 ----
+endif::[]
+
+ifdef::satellite[]
```

Comment:
why the difference between Foreman/Satellite? The only difference is EL/RHEL in the "After the system reboots" line, which is the only line that should be conditioned.

Reply:
Changed it.